components/microsoft_outlook/microsoft_outlook.app.mjs

```diff
@@ -1,11 +1,72 @@
+import axios from "axios";
+
 export default {
   type: "app",
   app: "microsoft_outlook",
-  propDefinitions: {},
+  propDefinitions: {
+    recipients: {
+      label: "Recipients",
+      description: "Array of email addresses",
+      type: "string[]",
+      optional: false,
```

Comment:
`optional: false,` can be removed because the propery is mandatory by default. [The docs here](https://pipedream.com/docs/v1/components/api/#general)